src/BlockchainWriter/BlockInfoDAO.ts

```diff
@@ -0,0 +1,26 @@
+import { LightBlock } from 'Messaging/Messages'
+import { inject, injectable } from 'inversify'
+import { Collection } from 'mongodb'
+
+@injectable()
+export class BlockInfoDAO {
+  private readonly blockchainInfo: Collection
+
+  constructor(
+    @inject('BlockchainInfoCollection') blockchainInfo: Collection,
+  ) {
+    this.blockchainInfo = blockchainInfo
+  }
+
+  readonly start = async () => {
+    await this.blockchainInfo.createIndex({ height: 1, hash: 1 }, { unique: true })
```

Comment:
Just in case, won't we need to support finding by hash only too?

Reply:
yes. Will be adding that in a separate PR